public/styles/style.css

```diff
@@ -264,90 +262,180 @@ body {
       display: block;
       padding: 3px;
     }
-}
+
+    .card-favorites {
+      position: relative;
+      border: solid 0.1em var(--regent-blue-primary-color);
+      height: 60vh;
+      width: auto;
+      background-color: var(--white-primary-color);
```

Comment:
Wat zie je in een browser als custom properties niet worden ondersteund? Zorg dat je daar ook een oplossing voor hebt.